CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
-# Update [4.0.0]
+# Update [4.1.0]
```

Comment:
er det meningen at du fjerner 4.0.0?

Reply:
Tenkte litt feil på slutten her i går, det er vel mer korrekt å kalle denne her for en patch. Oppdatert nå